### DISCUSSION TITLE
Company logo wont save

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2493] [Help] [answered]
Hi Guys I keep getting an error when trying to save a new company logo. is this a known problem or something I am doing?
![image](https://user-images.githubusercontent.com/80058238/131375427-ff70115a-106b-4986-8939-765244281060.png)

**Comment by berma83** [comment]
Same error for me, version 0.9.116.

![image](https://user-images.githubusercontent.com/53205779/131389885-fd57e4d1-ab77-4fa0-b218-49455fe7c214.png)

**Comment by berma83** [comment, marked as answer]
Hi @AndrewJRamay

#2488 bug confirmed.

**Reply by KernOil** [reply]
@berma83  that will do it thank you, sir. must have missed that in my search.

**Comment by mslourens** [comment]
I just pushed a fix for #2488.

**Reply by shogunpurple** [reply]
@AndrewJRamay no - mslourens is one of our very kind contributors outside of budibase. 

Just a gentle reminder - the team is working as hard as they can on delivering features and improving budibase. You raised an issue for https://github.com/Budibase/budibase/issues/2435, we are aware of it and we will fix it as soon as possible. 
 
Please refain from tagging members of the team directly, or copying and pasting issue links across different discussions to try and hurry issues along. Budibase is free and open source, which of course means we are open to contributions also. If a fix has been "plaguing" your application, we ask kindly that you have the patience for us to fix it. Thank you for understanding

**Reply by KernOil** [reply]
@shogunpurple  I appreciate all your team is doing! I know each of your members is working very hard! keep up the great work!
